pogobuf/pogobuf.client.js

```diff
@@ -804,7 +826,11 @@ function Client() {
 
     this.maxTries = 5;
     this.mapObjectsThrottlingEnabled = true;
-    this.mapObjectsMinDelay = DEFAULT_MAP_OBJECTS_DELAY * 1000;
+    this.mapObjectsMaxDelay = DEFAULT_MAP_OBJECTS_DELAY * 1000;
+    this.mapObjectsMinDistance = DEFAULT_MAP_OBJECTS_DISTANCE;
+    this.interactionThrottlingEnabled = true;
```

Comment:
If I remember correctly, you can do some actions outside told max distance, but it increases the possibility of getting the account banned.